include/iceflow/producer-tlv.hpp

```diff
@@ -318,7 +316,7 @@ class ProducerTlv {
                                   manifestDataBlock.type());
       }
 
-      manifestDataPacket.setFreshnessPeriod(ndn::time::seconds(10));
```

Comment:
Maybe we could create a constant for the freshness period to make it more clear why that value has been chosen?

Reply:
We have to discuss this.